docs/interchain-staking.md

```diff
@@ -0,0 +1,178 @@
+
+Instead of distributing rewards on the child chain, the rewards collected for each block produced should be transferred to the parent chain at some interval. It may be a regular interval or a user initiated IBC packet, but would be able be similar to an ics-20 token transfer from the distribution module account of the child chain to the distribution module account of the parent chain. The difference between a standard ics-20 transfer being that the parent chain distribution module account needs to know which chain the rewards came from and over which period they were collected. This information will be necessary to perform the parent chain distribution module's responsibility of distributed rewards to delegators and validators on a per chain basis informed by the Chain Registry.
+
+
+### Evidence
+
+In a single chain system, a validator may misbehave in various ways that result in the stake attached to that validator being slashed. This can occur automatically within the state machine or only after evidence of the misbehaviour has been collected and submitted to the chain. In a scenario where the validator set for a child chain is secured by stake that exists only on the parent chain, the evidence of misbehaviour needs to be submitted to the parent chain, where the tokens at stake are able to be slashed. Similar to a single chain system, this may take place automatically or only with the manual submission of evidence. If it were to occur automatically it would mean an outgoing IBC packet could be automatically generated but would still need to be manually relayed to the parent chain where punishment could follow. The more manual scenario would mean that the misbehaviour on the child chain results in evidence which is submitted directly to the parent chain, or submitted to the child chain which if successfully processed would result in an outgoing IBC packet containing the instruction to slash at the level of the parent chain.
+
+Either way comes with the question of whether it should be the parent chain that is verifying the evidence of slashable events, or simply trusting the child chain's commands to slash validators at the level of the parent. While there may be the ability to both, trusting the child chain makes the job of the parent chain much easier. In that scenario the parent chain doesn't need to have knowledge of the logic contained within the child chain that may be necessary for determining whether a slashable offense took place. Trusting the child chain to enforce its own rules puts a lot of trust in the child chain to not abuse the position and submit a wave of slashable commands that may reduce a validator to nothing. However this is ultimately the responsibility of the validator to ensure that the slashable risk of validating on a child chain is worth the expected rewards. This includes determining whether the state machine of the child chain includes logic that the validator is comfortable with.
```

Comment:
Note this means that a validator's stake is slashable by the weakest validator set it participates in. This is an important point to discuss and decide on as it really changes the calculus for validators on whether they want to participate in Interchain staking

Reply:
Note that it is not only about trusting state machine, as evidence can be blocked at the tendermint level (by censorship). It might be very hard for validator to compute the risk just by looking at validator set.

Reply:
this has always been my understanding altho putting limits to the amount a validator can be slashed or forcing them into jail could be remedies for runaway slashing events?